### PR TITLE
Update angular schematics to 16.2.10

### PR DESCRIPTION
**Pull request by tomodasheesh**

### CODE REVIEW

**Review by GoodDayForSurf — DISMISSED** [review]

File: packages/devextreme-cli/src/applications/application.angular.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const latestVersions = require('../utility/latest-versions');
 const { extractToolingVersion, toolingVersionOptionName } = require('../utility/extract-tooling-version');
 const schematicsVersion = latestVersions['devextreme-schematics'] || 'latest';
 
-const minNgCliVersion = new semver('12.0.0');
+const minNgCliVersion = new semver('16.2.10');
```
Comment (dxvladislavvolkov) [inline packages/devextreme-cli/src/applications/application.angular.js:13]: Change version to 16.0.0

File: packages/devextreme-schematics/src/add-layout/schema.json [hunk 2]
```diff
@@ -49,7 +49,7 @@
     },
     "globalNgCliVersion": {
       "type":"string",
-      "default": "12.2.0"
+      "default": "^16.2.10"
```
Comment (dxvladislavvolkov) [inline packages/devextreme-schematics/src/add-layout/schema.json:52]: Change to 16.2.0

File: packages/devextreme-schematics/src/add-app-template/schema.json [hunk 3]
```diff
@@ -43,7 +43,7 @@
     },
     "globalNgCliVersion": {
       "type":"string",
-      "default": "^12.0.0"
+      "default": "^16.2.10"
```
Comment (dxvladislavvolkov) [inline packages/devextreme-schematics/src/add-app-template/schema.json:46]: Also change version

File: packages/devextreme-schematics/src/add-layout/index_spec.ts [hunk 4]
```diff
@@ -25,27 +25,30 @@ describe('layout', () => {
...
     name: 'workspace',
-    version: '6.0.0'
+    version: '16.2.10'
   };
 
   const options: any = {
     layout: 'side-nav-outer-toolbar',
     resolveConflicts: 'override',
-    globalNgCliVersion: '^12.2.0'
+    globalNgCliVersion: '^16.2.10'
```
Comment (dxvladislavvolkov) [inline packages/devextreme-schematics/src/add-layout/index_spec.ts:34]: And here

**Review by dxvladislavvolkov — APPROVED** [review]

**Review by GoodDayForSurf — APPROVED** [review]

**Pull request merged**